apps/system/utils/README.md

```diff
@@ -183,6 +179,16 @@ PID | Pri |  Snap ticks  |
  * Snapshot interval : 10s (1000 ticks)
 --------------------------------------------------
 ```
+### How is the cpuload calculated
+We define a timeconstant (CONFIG_SCHED_CPULOAD_TIMECONSTANT), to keep a limit on total cpuload.
+We store "total ticks" and "ticks respective to a thread for specific core" after every systick interrupt.
+thread_n load on cpu_i = (thread_n load in cpu_i) / (total load in cpu_i)
+average thread_n load = (thread_n load in all cpu) / (total load in all cpu)
```

Comment:
`average thread_n load = (thread_n load in all cpu) / (total load in all cpu)`
Change to
`average thread_n load = (Sum of thread_n load in each cpu) / (Number of cpus)`